### DISCUSSION TITLE
[PorousFlow]__Debugging

### DISCUSSION THREAD

**Discussion by Joseph-0123** [idaholab/moose#23650] [Q&A Modules: General] [answered]
**Dear all,

When I use the PorousFlow module running my input, I get the following errors.** 

_"YOUR APPLICATION TERMINATED WITH THE EXIT STRING: Segmentation fault (signal 11)
This typically refers to a problem with your application. Please see the FAQ page for debugging suggestions"_

**So I follow the [Debugging document](https://mooseframework.inl.gov/application_development/debugging.html), and I get the following tips:**

Thread 1 "porous_flow-dbg" received signal SIGSEGV, Segmentation fault.
0x0000155554ac121c in PorousFlowMaterial::sizeNodalProperties (this=0x262fa40) at /home/projects/moose/modules/porous_flow/src/materials/PorousFlowMaterial.C:147
147         props_old[prop_id]->resize(new_size);
Missing separate debuginfos, use: yum debuginfo-install keyutils-libs-1.5.10-6.el8.x86_64 krb5-libs-1.18.2-9.el8_4.x86_64 libX11-1.6.8-4.el8.x86_64 libXau-1.0.9-3.el8.x86_64 libblkid-2.32.1-27.el8.x86_64 libcom_err-1.45.6-2.el8_4.x86_64 libmount-2.32.1-27.el8.x86_64 libpciaccess-0.14-1.el8.x86_64 libselinux-2.9-5.el8.x86_64 libtirpc-1.1.4-4.el8.x86_64 libunwind-1.3.1-3.el8.x86_64 libuuid-2.32.1-27.el8.x86_64 libxcb-1.13.1-1.el8.x86_64 libxml2-2.9.7-9.el8_4.2.x86_64 openssl-libs-1.1.1g-16.el8_4.x86_64 pcre2-10.32-2.el8.x86_64 systemd-libs-239-45.el8_4.15.x86_64 xz-libs-5.2.4-4.el8_4.x86_64 zlib-1.2.11-18.el8_4.x86_64

**How can I solve this problem? Thanks for your help in advance.**

J

**Comment by cpgr** [comment]
This is a pretty low-level material in porous flow. Can you post a minimal example that demonstrates this? 

Have you made your own material or are you using just the standard materials available in the module?

**Comment by cpgr** [comment, marked as answer]
For those reading along, this should be fixed by #23672

**Reply by Joseph-0123** [reply]
> For those reading along, this should be fixed by #23672

Thanks a lot, Chris! It works!
